client/src/components/Query/Filter/Generic.jsx

```diff
@@ -234,18 +234,21 @@ class GenericFilter extends React.Component {
   }
 
   render() {
-    const { allOptions } = this.state;
+    const { allOptions, draft } = this.state;
+    const { operand } = draft;
     const { config } = this.props;
+
     return (
       <Filter
         {...this.props}
         config={config}
         type={FILTER_TYPE_GENERIC}
         editor={this.getEditor()}
+        operand={operand}
```

Comment:
@FrancisRenaud No need to pass operand props since it is already contained within the data prop under data.operand for the relevant operators (should ba available in the draft too).

Reply:
Well it seems that the info is is the state of the Generic or Specific filter and is not passed to the Filter component, which needs it.

Reply:
For now I'll pass the `draft` to the `Filter` component. This is where the newly changed operand value resides.